ocs_ci/ocs/pgsql.py

```diff
@@ -258,6 +278,68 @@ def validate_pgbench_run(self, pgbench_pods):
             all_pgbench_pods_output.append(pg_output)
         return all_pgbench_pods_output
 
+    def get_nodes(self, pod_name, all_nodes=False):
+        """
+         Get nodes that contain a specific pod
+
+        Args:
+            pod_name (str)
+            all_nodes (bool):
+                True-return all the nodes that contain postgres pod
+                False-return 1 node that contain postgres pod
+        Returns:
+            list: Cluster node OCP objects
+
+        """
+        if pod_name == 'osd':
+            pods_obj = self.pod_obj.get(selector=constants.OSD_APP_LABEL, all_namespaces=True)
+        elif pod_name == 'postgres':
+            pods_obj = self.pod_obj.get(selector=constants.PGSQL_APP_LABEL, all_namespaces=True)
```

Comment:
what about other conditions? if the pod name is neither of 'osd' or 'postgres' then we need an `else` statement I think